virttest/qemu_vm.py

```diff
@@ -2471,6 +2471,10 @@ def sort_key(dev):
             else:
                 attr_info = ["mlock", params["realtime_mlock"], bool]
                 add_qemu_option(devices, "realtime", [attr_info])
+        if params.get("cpu-pm"):
+            if devices.has_option("overcommit"):
+                attr_info = ["cpu-pm", params["cpu-pm"], bool]
+                add_qemu_option(devices, "overcommit", [attr_info])
```

Comment:
What if `realtime_mlock` is defined together with `cpu-pm` and `overcommit`? You will add two `overcommit` string devices, is that expected?

Reply:
Yes, they can be defined together.

Reply:
@huangyum That would be `-overcommit realtime_mlock=xx -overcommit cpu-pm=xx`, are you sure that's correct command line?

Reply:
Yes.

Reply:
Shouldn't it be `-overcommit mem-lock=***,cpu-pm=***`  ?   @huangyum

Reply:
@yanan-fu Both of the usage are working well. Since libvirt doesn't supported cpu-pm option yet, I prefer the usage in qemu help. Thanks.

Reply:
@huangyum I reserve my opinion for the format.
You are right, the qemu-kvm man page use the format you added, however they are still two options for `overcommit`:
```
/usr/libexec/qemu-kvm --overcommit \?
overcommit options:
  cpu-pm=<bool (on/off)>
  mem-lock=<bool (on/off)>
```

Reserve my opinion, while not against yours, thanks~